Classes/Backend/DynamicFlexForm.php

```diff
@@ -323,7 +320,10 @@ protected function recursivelyEvaluateClosures(array $dataStructureArray, array
      */
     protected function getCache()
     {
-        return GeneralUtility::makeInstance(CacheManager::class)->getCache('flux');
+        if ($this->cache !== null) {
```

Comment:
I don't think I get what you want to memoize here. The values or a single cache instance on a static property?